src/bundle/Resources/public/scss/_login.scss

```diff
@@ -11,13 +11,17 @@
         color: $ez-color-warning-dark;
     }
 
+    &__hyperlink {
+        color: #0005a6;
```

Comment:
I'm not sure, shouldn't we use it by variable? @dew326

Reply:
This is an acceptable exception. This color is specific only for this view.